automation/test.sh

```diff
@@ -125,6 +125,10 @@ fi
 # Give the nodes enough memory to run tests in parallel, including tests which involve fedora
 export KUBEVIRT_MEMORY_SIZE=${KUBEVIRT_MEMORY_SIZE:-11264M}
 
+# All lanes enable CPUManager and wait for it to appear as part of the test suite setup.
+# That process can take more than a minute. Enabling it from the start will free that time.
+export FEATURE_GATES=CPUManager
```

Comment:
I think we should check if the `FEATURE_GATES` env variable is set and if it is - append CPUManager to it. 

As it is currently we could overwrite FEATURE_GATES that may be set for specific lanes like here - https://github.com/kubevirt/project-infra/blob/c8f48bf4da6907aa1dc0ca0e9c8d680709293c0b/github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml#L2050C15-L2051C32

Reply:
Woops yeah, nice catch!